src/functions.php

```diff
@@ -202,8 +202,11 @@ function is_string_array($input) {
  *
  * @internal
  * @see https://bugs.php.net/bug.php?id=49664
+ *
  * @param mixed $element Value to be copied
+ *
  * @return mixed
+ * @throws \ReflectionException
```

Comment:
Add `use ReflectionException` up top (after ReflectionClass) and change this to `@throws ReflectionException`. For consistency, we try to `use` all classes (top-level or not) that are referenced in each file.